js/docs.js

```diff
@@ -292,6 +292,26 @@ window.onload = function() {
     var group = $(this).attr('data-group');
     $('.nav-tabs > li > a[data-group="'+ group +'"]').tab('show');
   })
+
+  // isArchive is set by logic in archive.js
+  if ( isArchive == false ) {
+    console.log("Showing content that should only be in the current version.");
+    // Hide elements that are not appropriate for archives
+    // PollDaddy
+    $('#ratings-div').css("display","block");
+    console.log("Ratings widget shown.");
+    // Archive drop-down
+    $('.ctrl-right .btn-group').css("display","block");
+    console.log("Archive widget shown.");
+    // Swarch
+    $('.search-form').css("display","block");
+    console.log("Search widget shown.");
+    // Page edit link
+    $('.feedback-links li').first().css("display","block");
+    console.log("Page edit link shown.");
+  } else {
+    console.log("Keeping non-applicable elements hidden.");
+  }
```

Comment:
An alternative could be to have a css class like;

```css
.is-archive .hide-on-archive { display: none }
```

Then, add the `.is-archive` on a top-level element on the page (`<body class="is-archive">` or `<html class="is-archive">`.

Instead of finding all the elements that should _possibly_ be hidden, you can just remove the `.is-archive` class from the body, and those elements become visible (reducing the risk that elements may/may not be present in a specific version of the archive, and reduces maintenance of the script)

Reply:
(each element that needs to be hidden should then have a `class="hide-on-archive"` class)

Reply:
I don't really know enough to implement this. Feel free to commit into this PR if you want to try it, after I address the other feedback.